docs/usage/model_config.md

```diff
@@ -28,6 +28,9 @@ _(This script is complete, it should run "as is")_
 **`anystr_lower`**
 : whether to make all characters lowercase for str & byte types (default: `False`)
 
+**`anystr_upper`**
+: whether to make all characters uppercase for str & byte types (default: `False`)
+
```

Comment:
As you placed `upper` on top of the lower in other places of the patch, I would suggest moving this part on top of the `anystr_lower`

Reply:
adjust on 8aa6032